.github/ISSUE_TEMPLATE/api_deprecation.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Deprecated API
+about: Track the cleanup of a deprecated API
+title: ''
+labels: 'api deprecation'
+assignees: ''
+
+---
+
+## Deprecated API
+<!-- Describe the API along with its associated classes, interfaces, packages, etc. -->
+
+**Context**
+<!-- An explanation of why this API should be deprecated and ultimately removed -->
+
+**Approach**
+<!-- An explanation of the intended replacement/solution -->
+
+**Dependencies**
```

Comment:
Can we add explicit back-compatibility considerations here as its own separate line? The timeline of deprecation is driven by the needs around:
1. Persisted data compat. (forever compat. possibly can't move out)
2. Loader compat. (LTS support, current 5-6 months. Certianly longer once we get to 1.0)
3. Runtime (N-1) compat.

Reply:
Maybe we should just rename "Dependencies" to "Compatibility Concerns" and have the comment below list the points you gave?

Reply:
I see dependency as some other work that needs to be completed/solutioned before API can be fully removed after deprecation. I'd keep it separate from "Compatibility Concerns".